connect_sample/src/main/java/com/zendesk/sample/connect/FirebaseSetup.java

```diff
@@ -0,0 +1,30 @@
+package com.zendesk.sample.connect;
+
+import android.app.Application;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+
+class FirebaseSetup {
```

Comment:
It might be nice to collapse this into the `Global` class, so that there's only one file that must be updated with IDs/keys. It would be frustrating to first get a RuntimeException because you didn't add a Connect ID, then go add your Connect ID, then get another RuntimeException because you didn't add your Firebase details. RuntimeExceptions in general aren't much fun :D